WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1686,6 +1885,20 @@ private boolean hasCurrentTag() {
     }
 
     private void setCurrentTag(final ReaderTag tag) {
+        setCurrentTag(tag, false);
+    }
+
+    private boolean isValidTagForSharedPrefs(ReaderTag tag) {
```

Comment:
I'd also extract this one :-)

Reply:
yep. done here cb0b1590629e59f73babc5f04b9522100bf4d687